### PR TITLE
Add Babel instead of traceur

### PR DESCRIPTION
**Pull request by sthuck**
Jira task: 
NGG-228

 - projects who want to enable babel just need to add 'babelEnabled: true' in their options.

- Also, you need to add to .jshinrc the option "esnext": true. There is no way to send that option to jshint through the config, if a .jshinrc file is being used all options passed through grunt are ignored.

### CODE REVIEW

File: grunt-helpers/grunt-init-config.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = function (grunt, options) {
 		autoprefixer:           require('../grunt-sections/transform-css')(grunt, options).autoprefixer,
 		compass:                require('../grunt-sections/transform-css')(grunt, options).compass,
 		sass:                   require('../grunt-sections/transform-css')(grunt, options).sass,
-		traceur:                require('../grunt-sections/transform-js')(grunt, options).traceur,
+		babel:									require('../grunt-sections/transform-js')(grunt, options).babel,
```
Comment (shahata) [inline grunt-helpers/grunt-init-config.js]: somekind of whitespace issue here

File: grunt-helpers/grunt-load.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ module.exports = function (grunt, options) {
     plugins = plugins.concat(options.inline ? ['grunt-extract-styles', 'grunt-wix-inline'] : []);
     plugins = plugins.concat(options.svgFontName ? ['grunt-webfont'] : []);
     plugins = plugins.concat(options.autoprefixer ? ['grunt-autoprefixer'] : []);
-    plugins = plugins.concat(featureDetector.isTraceurEnabled() ? ['grunt-traceur-latest'] : []);
+    plugins = plugins.concat(options.babelEnabled && featureDetector.isBabelEnabled() ? ['grunt-babel'] : []);
```
Comment (shahata) [inline grunt-helpers/grunt-load.js]: why not check only `options.babelEnabled`? I think we can just remove `isBabelEnabled` from `featureDetector`

**Comment by shahata** [comment]
@sthuck nice work! except for two small comments I think this is ready to merge

**Comment by sthuck** [comment]
@shahata fixed those issues

**Comment by shahata** [comment]
merged! thanks!

**Pull request merged**